### ISSUE TITLE
Asuswrt-Merlin (armv7): nextdns updates from 1.39.4 to dev instead of 1.40.0

### ISSUE THREAD

**Issue by pro-sumer** [nextdns/nextdns#864]
**Context**
 - CLI Version: 1.39.4
 - Platform: Asuswrt-Merlin (armv7)

`nextdns` updates from 1.39.4 to `dev` instead of 1.40.0

```
INFO: OS: asuswrt-merlin
INFO: GOARCH: armv7
INFO: GOOS: linux
INFO: NEXTDNS_BIN: /jffs/nextdns/nextdns
INFO: INSTALL_RELEASE: 1.40.0
u) Upgrade NextDNS from 1.39.4 to 1.40.0
c) Configure NextDNS
r) Remove NextDNS
e) Exit
Choice (default=u): u
INFO: Upgrading NextDNS...
NextDNS installed and started using merlin init
u) Upgrade NextDNS from dev to 1.40.0
c) Configure NextDNS
r) Remove NextDNS
e) Exit
Choice (default=u): u
INFO: Upgrading NextDNS...
NextDNS installed and started using merlin init
u) Upgrade NextDNS from dev to 1.40.0
c) Configure NextDNS
r) Remove NextDNS
e) Exit
...
```

No matter how many times I choose `u` next; `nextdns` remains at `dev`:

```
➜ nextdns version
nextdns version dev
```

**Comment by bakerboy448** [comment]
Seems to be universal. Not only Merlin.

**Comment by rs** [comment]
The build seems to miss the version in the binary. We’ll look into that.

**Comment by rs** [comment]
Should be fixed with 1.40.1

**Comment by pro-sumer** [comment]
Indeed fixed after updating to 1.40.1 🎉

Thanks for fixing!
